src/index.ts

```diff
@@ -66,6 +70,12 @@ export default class TestingFarmAPI {
 
   async newRequest(request: NewRequest): Promise<NewRequestResponse>;
   async newRequest(request: NewRequest, strict: boolean): Promise<unknown>;
+  async newUnsafeRequest(request: NewUnsafeRequest, strict: boolean): Promise<unknown> {
```

Comment:
Please move this function outside of function signatures.

Also, I would suggest the following implementation instead:
```ts
async unsafeNewRequest(request: unknown): Promise<unknown> {
  return this.link.post('requests', request);
}
```